config/spryk/templates/Zed/Persistence/Mapper/ZedPersistenceMapper.php.twig

```diff
@@ -0,0 +1,81 @@
+<?php
+
+{{ include('Partials/license.twig') }}
+
+namespace {{ organization }}\Zed\{{ module }}\Persistence\Mapper;
+
+use Generated\Shared\Transfer\{{ domainEntity }}CollectionTransfer;
+use Generated\Shared\Transfer\{{ domainEntity }}Transfer;
+use Orm\Zed\{{ module }}\Persistence\Base\{{ tableName | camelCased }};
+use Propel\Runtime\Collection\ObjectCollection;
+
+class {{ module }}Mapper implements {{ module }}MapperInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\{{ domainEntity }}CollectionTransfer ${{ domainEntity | lcfirst }}CollectionTransfer
+     *
+     * @return \Orm\Zed\{{ module }}\Persistence\Base\{{ tableName | camelCased }}[]|\Propel\Runtime\Collection\ObjectCollection
+     */
+    public function map{{ domainEntity }}TransferCollectionTo{{ domainEntity }}EntityCollection(
+        {{ domainEntity }}CollectionTransfer ${{ domainEntity | lcfirst }}CollectionTransfer
+    ): ObjectCollection {
+        ${{ domainEntity | lcfirst }}EntityCollection = new ObjectCollection();
```

Comment:
Usually we use the object we want to fill in the result as the second argument of the method, wouldn't it be more correct to do the same here?

Reply:
I've just taken an example from
\Spryker\Zed\CompanyUser\Persistence\Mapper\CompanyUserMapper. I'll try to find out